editor/src/components/inspector/sections/component-section/data-picker-popup.tsx

```diff
@@ -60,6 +75,21 @@ export interface JSXOption {
 
 export type VariableOption = PrimitiveOption | ArrayOption | ObjectOption | JSXOption
 
+const DataPickerFilterOptions = ['all', 'preferred'] as const
+export type DataPickerFilterOption = (typeof DataPickerFilterOptions)[number]
+export function dataPickerFilterOptionToString(mode: DataPickerFilterOption): string {
+  switch (mode) {
+    case 'all':
+      return 'All'
+    case 'preferred':
+      return 'Preferred'
+    default:
+      assertNever(mode)
+  }
+}
+
+export const DataPickerPreferredAllAtom = atom<DataPickerFilterOption>('preferred')
```

Comment:
Why is this in an atom? Just to persist between showings of the inspector?

Reply:
yep to persist